packages/core/src/amazonqGumby/app.ts

```diff
@@ -27,6 +27,10 @@ export function init(appContext: AmazonQAppInitContext) {
         transformationFinished: new vscode.EventEmitter<any>(),
         processHumanChatMessage: new vscode.EventEmitter<any>(),
         linkClicked: new vscode.EventEmitter<any>(),
+        startHumanInTheLoopIntervention: new vscode.EventEmitter<any>(),
+        promptForDependencyHIL: new vscode.EventEmitter<any>(),
+        HILSelectionUploaded: new vscode.EventEmitter<any>(),
```

Comment:
why is HIL abbreviated only sometimes? also, `HILSelectionUploaded` does not follow the naming conventions.
```suggestion
        startHilIntervention: new vscode.EventEmitter<any>(),
        promptForDependencyHil: new vscode.EventEmitter<any>(),
        hilSelectionUploaded: new vscode.EventEmitter<any>(),
```